source/custom-output-cli.txt

```diff
@@ -0,0 +1,250 @@
+.. _go-template-output: 
+
+=============================
+Customize the {+atlas-cli+} Output
+=============================
+
+.. default-domain:: mongodb
+
+You can customize the {+atlas-cli+} output fields and format using a ``Go`` 
+template or a JSON path.
+                    
+Go templates
+------------
+                    
+You can specify the Go template with the command for a simple output 
+or through a separate file for a complex output. For information on the 
+template, see `Package template <https://golang.org/pkg/text/template/>`__. 
```

Comment:
Simplifying text where it's a little tough to parse and changing "the template" to "a template"
```suggestion
You can specify a Go template within any {+atlas-cli+} command 
or through a separate file. To learn more about 
Go templates, see `Package template <https://golang.org/pkg/text/template/>`__. 
```